docs/antora-playbook.yml

```diff
@@ -0,0 +1,12 @@
+site:
```

Comment:
Rename this file?

Reply:
I'm not sure we can rename antora-playbook file, that's the common standard name used for most antora docs for building the site